pkg/monitor/querier.go

```diff
@@ -118,6 +120,11 @@ func (monitor *agentMonitor) GetLocalPodNum() int32 {
 	return int32(monitor.interfaceStore.GetContainerInterfaceNum())
 }
 
+// GetInterfacesByType gets the interface information by type.
+func (monitor *agentMonitor) GetInterfacesByType(interfaceType interfacestore.InterfaceType) []*interfacestore.InterfaceConfig {
```

Comment:
This command has nothing to do with monitor. Please do not add more CLI/API implementation to monitor. You can either call interfaceStore from apiserver directly, or if really needed add a new wrapper package.

Reply:
I am not sure if new framework changes this. Can we get latest info from apiserver @weiqiangt ?

Reply:
I think to call interfaceStore from apiserver directly, at least we need a "GetInterfaceStore" function which is not specific for cli. Code is updated accordingly.

Reply:
Could we pass InterfaceStore to apiserver/handler, just like we pass it to monitor/querier?